### DISCUSSION TITLE
Was für eine Technologie nutzt ihr?

### DISCUSSION THREAD

**Discussion by chrissharkman** [christianheimann/swissdigitalinfrastructure#4] [Q&A] [answered]
Wurde bei euch bereits eine konkrete Technologie getestet oder in produkiver Umgebung genutzt? Danke dass du deine Erfahrungen hier teilst.

**Comment by christianheimann** [comment, marked as answer]
Wir haben schon einmal HTML genutzt für eine Webseite. War klasse!
